plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -82,25 +96,26 @@ export default class NodesTable extends React.Component<
   retrieveSortFunction(sortColumn: string): SortFunction<Node> {
     switch (sortColumn) {
       case "host":
-        return ipSorter;
+        return sorter(compareByIp);
       case "type":
-        return typeSorter;
+        return sorter(compareByType);
       case "region":
-        return regionSorter;
+        return sorter(compareByRegion);
       case "zone":
-        return zoneSorter;
+        return sorter(compareByZone);
       case "health":
-        return healthSorter;
+        return sorter(UnitHealthUtil.getHealthSortFunction);
       case "tasks":
-        return tasksSorter;
+        return sorter((a, b) => getTasks(a) - getTasks(b));
```

Comment:
Non-blocking, but I think by removing some code, we might be making this harder to understand.